docs/source/configure/index.rst

```diff
@@ -146,6 +146,8 @@ Environment-specific parameters
     }
   ]
 
+- ``max_past_created_time`` is an optional parameter specifying transaction how many hours in the past since current peer time transaction's `created_time` can be setted. The default value is `"24"` hours. The value should be the same in a whole iroha network.
```

Comment:
```suggestion
- ``max_past_created_time`` is an optional parameter specifying how many hours in the past since current time (measured on the peer) can the transaction's `created_time` can be set. The default value is `"24"` hours. This value must be the same on all peers, otherwise it can silently cause the network to stop producing blocks. 
```